nn.py

```diff
@@ -522,16 +518,21 @@ def __init__(self, num_models, dropout, device):
             dropout    = [float] prob of dropping inter-group weight gradient
             device     = [torch.device] device to put the models and data on
         """
-        global num_groups
+        global num_const
 
         super(Model, self).__init__()
 
-        min_model_groups = num_groups // num_models
-        num_extra_groups = num_groups % num_models
+        # make linear layer for embeddings constant per store-item group
+        self.fc = nn.Linear(ForecastDataset.num_const, num_const).to(device)
```

Comment:
Is `ForecastDataset.num_const` anders dan `num_const`? Ik dacht je de `global num_const` in de `ForecastDataset` hebt gestopt en dat je dan niet meer `global num_const` nodig hebt (i.e., ze zijn hetzelfde).

Reply:
Alle one-hot vectors bij elkaar zijn 114 elementen (`ForecastDataset.num_const`) en na de FC laag hebben we embeddings van 20 elementen (`nn.num_const`).